<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenProject</class>
 <widget class="QDialog" name="OpenProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>881</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="cmdProjectDel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdOpenProject">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>440</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Selected Project</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lstProjects">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>851</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbFilter">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtFilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdFilter">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>60</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Filter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
